feature_details.md

```diff
@@ -0,0 +1,174 @@
+    * `metadataOnly` : A boolean indicating whether the UA should skip capturing the element snapshot, defaults to false. This is purely a performance optimization when a visually identical element is present in both Documents, which would be a common scenario for paired transitions.
+
+* The `handleTransition` event provides the information for a cached element from the previous Document using a new interface called `SharedElement`. It has a `record<DOMString, any> getPropertyMap()` API to retrieve the `propertyMap` value above. This `SharedElement` can then be referenced in `ElementTransition` for declaring exit and/or paired transitions.
+
+#### Hierarchical Properties
+The web platform has multiple concepts which define the coordinate space in which the animatable properties referenced above should be interpolated : [Containing Block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block), [3D rendering context](https://drafts.csswg.org/css-transforms-2/#3d-rendering-contexts); the order in which the elements are stacked : [Stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context); and the element to which an effect like backdrop-filter is applied : [Backdrop Root](https://drafts.fxtf.org/filter-effects-2/#BackdropRoot). All these concepts are defined based on where an element fits in the DOM hierarchy. These are tricky to reason about in the context of a transition because :
+
+* It is desirable to change the ancestor for these concepts during the transition. For example, an element may be clipped by the viewport instead of an ancestor element as it animates to exit the screen.
+
+* Paired transitions are executed by creating a combined image for elements in the two trees on which the animated properties are applied. Should the transition switch to using a parent stacking context for the shared element in the new Document when a transition starts?
+
+The following steps define the behaviour of how content from the two Documents is drawn together during the transition. The aim is to avoid an element’s visual representation abruptly changing at transition start/end for common use-cases :
+
+* We assume that the dimensions for the initial containing block for both Documents is the same. If this changes, for example because the user resizes the browser window before the transition finishes, the transition is aborted.
+
+* During the transition, the UA creates a new root element (hidden from script), referred to as `TransitionRoot` which establishes the root stacking context and containing block. The background color for the `TransitionRoot` is the same as the background color for the root element in the new Document. By default, all shared elements (including root elements from both Documents) are direct descendants of the `TransitionRoot`.
```

Comment:
This seems to be describing a particular implementation, rather than effect. Or do you mean to say that this is an example of how it might be implemented?

Reply:
Hmmm, I wasn't trying to push for a specific implementation. My aim here was to explain how elements are drawn during a transition using web platform concepts so we can understand the space in which properties like 'opacity' or 'transform' exist.

For example, the transition could animate the transform for the root element of a Document. In the default case this transform would apply to its entire subtree. But during the transition, the shared elements from this Document are "detached" from their root element. The concept of a temporary "root" element as described here was a way to explain the relationship between a Document's root element and its shared elements during the transition.